torch/ao/quantization/fx/_lower_to_native_backend.py

```diff
@@ -118,16 +129,23 @@ def special_pattern_replacement(model: QuantizedGraphModule) -> QuantizedGraphMo
                         dq_node.target = ref_node
 
                     # remove dq node:
-                    dq_node_input = dq_node.args[0]
-                    dq_node.replace_all_uses_with(dq_node_input)
-                    model.graph.erase_node(dq_node)
+                    if isinstance(dq_node, Node):
+                        dq_node_input = dq_node.args[0]
+                        dq_node.replace_all_uses_with(dq_node_input)
+                        model.graph.erase_node(dq_node)
+                    elif isinstance(dq_node, (tuple, list)):
+                        for dn in dq_node:
+                            dn_input = dn.args[0]
+                            dn.replace_all_uses_with(dn_input)
+                            model.graph.erase_node(dn)
```

Comment:
Maybe we can do something like the following to reduce the duplicate code?
```
dq_nodes = []
if isinstance(dq_node, Node):
    dq_nodes = [dq_node.args[0]]
elif isinstance(dq_node, (tuple, list)):
    dq_nodes = dq_node.args[0]

for dn in dq_nodes:
    ... # replace the node from the graph
```

Reply:
makes sense, will do